src/conversion/modelconversiondispatcher.js

```diff
@@ -188,7 +198,9 @@ export default class ModelConversionDispatcher {
 				range: itemRange
 			};
 
-			this._testAndFire( 'insert', data, consumable );
+			const name = item.name || '$text';
+
+			this._testAndFire( 'insert', consumable, data, name );
```

Comment:
This is like this to support `removeAs` parameter in remove conversion (for rename purposes).